### PR TITLE
feat: enleap launchpad

### PR DESCRIPTION
**Pull request by iqbalutomo**

### CODE REVIEW

File: components/Launchpad/LaunchpadStats.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		setShowTooltip(true)
+		getMintDuration()
+	}, [project])
+
+	const getMintDuration = () => {
+		const startDate = new Date(project?.mint_details[0].started_at)
+		const endedDate = new Date(project?.mint_details[0].ended_at)
+		const diff = endedDate.getTime() - startDate.getTime()
+		setMintDuration(Math.floor(diff / 1000 / 60 / 60))
+	}
```
Comment (ahnafalfariza) [inline components/Launchpad/LaunchpadStats.js:23]: `mintDuration` kira2 perlu masuk useEffect dan state ga yah? soalnya best practicenya [gini](https://beta.reactjs.org/learn/you-might-not-need-an-effect#how-to-remove-unnecessary-effects)
```js
const getMintDuration = (startDateTime, endDateTime) => {
  const startDate = new Date(startDateTime)
  const endedDate = new Date(endDateTime)
  const diff = endedDate.getTime() - startDate.getTime()
  setMintDuration(Math.floor(diff / 1000 / 60 / 60))
}

const LaunchpadStats = ({ project, isEnded, isEndedComing, isValidating }) => {
	const mintDuration = getMintDuration(project?.mint_details[0].started_at, project?.mint_details[0].ended_at)
  
  // more code
}
```

File: components/Launchpad/TimeLaunchpad.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	const [days, setDays] = useState('-')
+	const [hours, setHours] = useState('-')
+	const [minutes, setMinutes] = useState('-')
+	const [seconds, setSeconds] = useState('-')
+	const [isEndedTime, setIsEndedTime] = useState(false)
+
+	useEffect(() => {
+		countDownTimer()
+	}, [isEndedTime])
+
```
Comment (ahnafalfariza) [inline components/Launchpad/TimeLaunchpad.js:13]: gua ada bikin juga component [countdown](https://github.com/ParasHQ/paras-landing/blob/develop/components/Common/CountdownSimple.js) bal, gua make statenya cuman satu

Comment (ahnafalfariza) [reply components/Launchpad/TimeLaunchpad.js:13]: tapi ini fyi aja sih

File: pages/launchpad/[launchpad_id]/project/[project_id].js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+			notFound: true,
+		}
+	}
+
+	return {
+		props: {
+			project: resp.data,
+		},
+	}
+}
```
[the remark this answers is not in the document]
Comment (iqbalutomo) [reply pages/launchpad/[launchpad_id]/project/[project_id].js:208]: ok hu done

File: pages/launchpad/[launchpad_id]/project/[project_id].js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+	const [tabActive, setTabActive] = useState('story')
+	const [isEndedTime, setIsEndedTime] = useState(false)
+	const router = useRouter()
+
+	const fetchData = (url) => axios(url).then((res) => res.data)
+
+	const { data, isValidating } = useSWR(
+		`${process.env.V2_API_URL}/launchpad/${router.query.launchpad_id}/project/${router.query.project_id}`,
+		fetchData
+	)
```
[the remark this answers is not in the document]
Comment (iqbalutomo) [reply pages/launchpad/[launchpad_id]/project/[project_id].js:28]: i see, pake fallbackData ya hu, okok hu

File: components/Launchpad/LaunchpadStats.js (hunk 1 above)
Comment (iqbalutomo) [reply components/Launchpad/LaunchpadStats.js:23]: mantap, noted hu

**Pull request merged**